package.json

```diff
@@ -163,7 +163,7 @@
     "typescript": "^2.5.2",
     "url-loader": "^0.5.7",
     "webpack": "3.8.1",
-    "webpack-dev-server": "2.9.4",
+    "webpack-dev-server": "2.7.1",
```

Comment:
had to downgrade this for now (seeing the same issue reported in https://github.com/webpack/webpack-dev-server/issues/1101)